oteapi/strategies/mapping/__init__.py

```diff
@@ -0,0 +1 @@
+from .mapping import MappingStrategy, MappingStrategyConfig
```

Comment:
```suggestion
```
These init files should not import anything. The strategies are retrieved through the `oteapi.plugins` module.